test_utils/cleanup.go

```diff
@@ -0,0 +1,46 @@
+package testing_init
+
+import (
+	"testing"
+
+	"github.com/port-labs/port-k8s-exporter/pkg/port/blueprint"
+	"github.com/port-labs/port-k8s-exporter/pkg/port/cli"
+	"github.com/port-labs/port-k8s-exporter/pkg/port/page"
+	"github.com/stretchr/testify/assert"
+)
+
+func CheckResourcesExist(portClient *cli.PortClient, t *testing.T, blueprints []string, pages []string) {
```

Comment:
I believe you can have a base function and call it twice, with a flag of "exists". The functions are exactly the same except assert.Nil vs assert.NotNil

Reply:
Nice catch, let me work it through